### PR TITLE
platform: remove support for cAVS 1.5 platforms

### PR DESCRIPTION
**Pull request by lyakh**
Remove all support for cAVS 1.5 platformsm including Apollo Lake, Sky Lake, Kaby Lake, Broxton and Gemini Lake, they aren't supported any more.

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

**Comment by lyakh** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
@marc-hb does CI need a script update for Jenkins FW build ?

**Comment by marc-hb** [comment]
> @marc-hb does CI need a script update for Jenkins FW build ?

Yes it looks like it but I don't know where sorry.
https://sof-ci.01.org/sofpr/PR7139/build4072/build/index.html

Different topic: to fix the `dai_config_get` + IPC4 failures in https://github.com/thesofproject/sof/actions/runs/4233757417/jobs/7355127923 simply do an empty `git commit --amend -C HEAD` and force push. Fixed in #6939

File: tools/sof_ri_info/sof_ri_info.py [hunk 1]
```diff
@@ -31,39 +31,6 @@
 # - add a new one as array of bytes and append entry to KNOWN_KEYS below.
 #   you can use --full_bytes to dump the bytes in below format
 
-APL_INTEL_PROD_KEY = bytes([0x1f, 0xf4, 0x58, 0x74, 0x64, 0xd4, 0xae, 0x90,
```
Comment (marc-hb) [inline tools/sof_ri_info/sof_ri_info.py:34]: Please don't remove this key (and restore the older ones in different PR, sorry I missed this), this is going to break sof-bin tests for no good reason. This is just a pure constant with zero actual logic depending on it.

**Review by marc-hb — CHANGES_REQUESTED** [review]

File: tools/sof_ri_info/sof_ri_info.py (hunk 1 above)
Comment (lyakh) [reply tools/sof_ri_info/sof_ri_info.py]: @marc-hb ok, removed. It wasn't changed in earlier PRs - BYT / BDW didn't use signed firmware AFAIK

**Review by lgirdwood — APPROVED** [review]

**Review by marc-hb — COMMENTED** [review]
One hopefully last request: can you remove the topologies in a separate commit? We're still discussing how topologies will be packaged and released. This should be a very quick change for you: simply move all the changes in `tools/` to a separate commit.

**Comment by lyakh** [comment]
> One hopefully last request: can you remove the topologies in a separate commit? We're still discussing how topologies will be packaged and released. This should be a very quick change for you: simply move all the changes in `tools/` to a separate commit.

@marc-hb why? What difference does it make for *this* PR?

**Comment by marc-hb** [comment]
Never mind, I forgot other topologies were already removed like this.

**Pull request merged**